### PR TITLE
Add documentation and scripts generation of SSP markdown

### PR DESCRIPTION
**Pull request by rahearn**
Addresses https://github.com/GSA-TTS/devtools-program/issues/64

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 # Want to help us make this template better? Share your feedback here: https://forms.gle/ybq9Krt8jtBL3iCk7
 
-ARG PYTHON_VERSION=3.9.16
+ARG PYTHON_VERSION=3.9.19
```
Comment (pauldoomgov) [inline Dockerfile:9]: Would be nice to ratchet up to a newer version, but I won't block on this cause I am not some sort of jerk.

File: scripts/generate-ssp-markdown [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+set -e
+
+if [ "$1" = "" ]; then
+    echo "Usage: $0 PROFILE_NAME"
+    exit 1
+fi
+
+if [ ! -d profiles/$1 ]; then
```
Comment (pauldoomgov) [inline scripts/generate-ssp-markdown:10]: In case someone has a profile named "I like spaces"
```suggestion
if [ ! -d "profiles/$1" ]; then
```

**Review by pauldoomgov — APPROVED** [review]
One minor non-blocking suggestion. Other than that, LGTM!

File: Dockerfile (hunk 1 above)
Comment (rahearn) [reply Dockerfile:9]: Until https://github.com/oscal-compass/compliance-trestle/pull/1533 is merged and released, 3.9 is the most recent supported version.

**Pull request merged**